src/cl03/signature.rs

```diff
@@ -116,6 +148,28 @@ impl<CS: CLCiphersuite> Signature<CL03<CS>> {
         false
     }
 
+    pub fn disclose_selectively(&self, messages: &[CL03Message], a_bases: Bases, pk: &CL03PublicKey, unrevealed_indexes: &[usize]) -> (Vec<CL03Message>, Bases) {
```

Comment:
If I understood correctly, this function performs the Selective Disclosure of messages as the `proof_gen` function, but without hiding the signature behind a proof. Is that correct?

Reply:
That's correct. This function returns a modified version of messages and a_bases so to be able to verify the (non blind) signature without revealing each message signed. 
This function leverages the dlog as follows:

inputs:
- b0 = bases
- m0 = messages.

outputs:
- b1 = new bases
- m1 = new messages.

for each un-disclosed message:
- b1 = b0 ^ m0
- m1 = 1

Reply:
Clear, thank you!